spec/tail_spec.js

```diff
@@ -1,10 +1,38 @@
-var tail = require("../src/tail.js")
+import {tail} from '../src/tail'
 
 describe('tail', ()=> {
   it('returns the first element in an array', () => {
-    expect(tail([1,2,3,4,5])).toEqual([2,3,4,5])
-    expect(tail([2,3,4,5,])).toEqual([3,4,5])
-    expect(tail([3,4,5,])).toEqual([4,5])
-    expect(tail([{first: 'first'},{second: 'second'},{third: 'third'},{fourth: 'fourth'},{fifth: 'fifth'},])).toEqual([{second: 'second'},{third: 'third'},{fourth: 'fourth'},{fifth: 'fifth'}])
+    const array = [1,2,3,4,5]
```

Comment:
We can omit these variable declarations since they are only used once; may as well declare them inline.